### PR TITLE
HDS-1684-yarn-cleanup

### PR DESCRIPTION
**Pull request by mrTuomoK**
yarn.lock cleanup,
fix array index & classNames errors,
fix jsx-a11y eslint-rules,
update lerna,
update rimraf,
bump node to v16 in GH

## Description

Removes a lot of critical warnings by updating packages (lerna, rimraf), also bump Node on CI to v16, fixes some eslint errors

## Related Issue

[HDS-1684](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1684)

## Motivation and Context

Critical and severe security etc. warning handled by updating stuff.

## How Has This Been Tested?

Local machine, CI build

[HDS-1684]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-1684?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mikkojamG — APPROVED** [review]

File: packages/react/src/components/header/components/headerUniversalBar/HeaderUniversalBar.tsx [hunk 1]
```diff
@@ -48,7 +49,7 @@ export const HeaderUniversalBar = ({
   const childElements = getChildElementsEvenIfContainerInbetween(children);
 
   return (
```
Comment (NikoHelle) [inline packages/react/src/components/header/components/headerUniversalBar/HeaderUniversalBar.tsx:51]: Is this a bug fix? Seems unrelated to linting errors.

Comment (mrTuomoK) [reply packages/react/src/components/header/components/headerUniversalBar/HeaderUniversalBar.tsx:51]: Ah yes this is a bug fix after all 👍🏻

**Review by NikoHelle — APPROVED** [review]
